components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -69,10 +70,14 @@ class ToolbarPresenter(
         toolbar.setSearchTerms(session?.searchTerms ?: "")
         toolbar.displayProgress(session?.progress ?: 0)
         updateToolbarSecurity(session?.securityInfo ?: Session.SecurityInfo())
+        session?.let {
+            updateToolbarTrackingProtection(it)
+        }
```

Comment:
You also want to do something here for "no session" (maybe use OFF?) since that is a valid case (e.g. no open tab in RB) and you wouldn't want to just show whatever was shown previously.

Reply:
👍

Reply:
Fixed!